storagext/lib/src/runtime/client.rs

```diff
@@ -207,8 +258,8 @@ impl Client {
     }
 }
 
-impl From<OnlineClient<PolkaStorageConfig>> for Client {
-    fn from(client: OnlineClient<PolkaStorageConfig>) -> Self {
-        Self { client }
-    }
-}
+// impl From<OnlineClient<PolkaStorageConfig>> for Client {
+//     fn from(client: OnlineClient<PolkaStorageConfig>) -> Self {
+//         Self { client, legacy_rpc: LegacyRpcMethods::<_>::new(client.into()) }
+//     }
+// }
```

Comment:
?

Reply:
Oh yeah, I forgot to remove that -> didn't figure out a way on how to convert client from OnlineClient to LegacyRpcMethods.